scheduler/scheduler.go

```diff
@@ -245,7 +250,8 @@ func createResources(toCreate chan *ScheduledResource, created chan string, ccLi
 			}
 
 			for {
-				time.Sleep(time.Millisecond * 100)
+				log.Printf("Checking status for %s", r.Key())
+				time.Sleep(time.Millisecond * 1000)
```

Comment:
Seems to be only moving one `log.Printf` several lines down. IMO this would be better to do that in this PR without introducing sleep change...